tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/TypedescriptorTest.java

```diff
@@ -873,21 +873,10 @@ public void testFlattenedUnionMembers() {
         assertEquals(type.typeKind(), UNION);
 
         expandedMembers = ((UnionTypeSymbol) type).memberTypeDescriptors();
-        TypeSymbol member1 = expandedMembers.get(0);
-        TypeSymbol member2 = expandedMembers.get(1);
-
-        assertEquals(member1.typeKind(), TYPE_REFERENCE);
-        assertEquals(member2.typeKind(), TYPE_REFERENCE);
-
-        member1 = ((TypeReferenceTypeSymbol) member1).typeDescriptor();
-        expandedMembers = ((UnionTypeSymbol) member1).memberTypeDescriptors();
         assertEquals(expandedMembers.get(0).typeKind(), INT);
         assertEquals(expandedMembers.get(1).typeKind(), STRING);
-
-        member2 = ((TypeReferenceTypeSymbol) member2).typeDescriptor();
-        expandedMembers = ((UnionTypeSymbol) member2).memberTypeDescriptors();
-        assertEquals(expandedMembers.get(0).typeKind(), FLOAT);
-        assertEquals(expandedMembers.get(1).typeKind(), BOOLEAN);
```

Comment:
Why were these assertions deleted?

Reply:
Judging by the modified behaviour of this test, looks like this change is breaking the `memberTypeDescriptors()` API.

Reply:
Sorry, my bad. Ignore these comments. This is corrects the previously buggy `memberTypeDescriptors()` API since we have defined it to be returning the expanded members.